README.md

```diff
@@ -1,10 +1,16 @@
 # Dependencies
 
-- [`flutter`](https://flutter.dev/docs/get-started/install)
-- [`gomobile`](https://godoc.org/golang.org/x/mobile/cmd/gomobile)
-- Copy `env.sh.example` to `env.sh` and update your `PATH` variable to expose both flutter and go bin directories
 
-    ```export PATH="$PATH:/path/to/go/bin:/path/to/flutter/bin```
+- [flutter](https://flutter.dev/docs/get-started/install)
```

Comment:
Seems we lost the single ticks along the way on a lot of things here, mind throwing them back in?